app/namespaces/data_provider/data_provider_service.py

```diff
@@ -211,4 +212,8 @@ def get_all_filter_options() -> Dict[str, Any]:
             0].isoformat()
         options["updated_at"] = session.query(func.max(DashboardSource.publication_date))[0][0].isoformat()
 
+        # Get population group options
+        results = session.query().with_entities(PopulationGroupOptions.order, PopulationGroupOptions.name, PopulationGroupOptions.french_name)
+        options["population_group"] = [[result[1], result[2]] for result in sorted(results, key=lambda result: result[0])]
```

Comment:
Are we sure this sorting will work properly if the order is null? I think the behaviour should be that options with null order should just be shoved at the end.

Also can we put a comment describing what we are sorting by since it is not clear on first read.

Reply:
for sure, will add that comment. this goes back to my above comment, but can Order be null? if it is, i agree with your approach and ill update, but i thought we would always have order because its linked to the filter options